src/executor/mutate/DeleteExecutor.cpp

```diff
@@ -58,6 +58,9 @@ folly::Future<Status> DeleteVerticesExecutor::deleteVertices() {
     if (vertices.empty()) {
         return Status::OK();
     }
+    std::sort(vertices.begin(), vertices.end());
+    vertices.erase(unique(vertices.begin(), vertices.end()), vertices.end());
```

Comment:
Need not to sort the vids, can use set to dedup them.

Reply:
> Need not to sort the vids, can use set to dedup them.

if using std::set , still need to convert it from std::set to std::vector. and it's not a good idea to change the parameter type of storage clients.

Reply:
I think you could add a `dedup` executor directly.

Reply:
Agree to add dedup plan node in execution plan.